index.js

```diff
@@ -37,6 +56,18 @@
 
 
 // ***** Question 4 *****
+const sameSameButDifferent = (num, maybeNum) => {
+  if (num === maybeNum) {
+    console.log('same same')
+  } else if (num === parseInt(maybeNum)) {
+    console.log('same same (but different)')
+  } else {
+    console.log('different')
+  }
+}
+sameSameButDifferent()
```

Comment:
The logic for your conditionals looks good, but just like above, this function is meant to **return** something instead of `console.log`. Also, when you call the function like on line 68, you need to pass it two values for the `num` and `maybeNum` parameters.

```js
const sameSameButDifferent = (num, maybeNum) => {
  if (num === maybeNum) {
    return 'same same'
  } else if (num === parseInt(maybeNum)) {
    return 'same same (but different)'
  } else {
    return 'different'
  }
}
console.log(sameSameButDifferent(123, "123"))
```